### ISSUE TITLE
The VoiceOver is not announcing the name of the Font size edit box of the General preference window of the Xamarin workbooks.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#269]
### Steps to Reproduce

1. Launch Xamarin Workbook application-> Select Mac platform and create a new workbook.
Turn On the Voiceover.
2. Navigate to Xamarin Workbooks-> preferences->Hit On Enter key-> General tab is opened by default.
3. Hit on Tab key to navigate to the interactive elements of the general preference window-> 
4. Hit on Tab key to navigate to Font size edit box of the general preference window.
5. Observe the Voiceover announcement for this Font size edit box.

### Expected Behavior
The Voiceover should announce the name of the edit box, so user could understand the purpose of the edit box.

### Actual Behavior
The VoiceOver is not announcing the name of the Font size edit box.
The VoiceOver is announcing as "# content selected edit text, edit text".
Note: The VoiceOver is announcing the value of the edit box, but not the name of the edit box as shown in the attachment.

### User Impact:
The VoiceOver user won’t be able to know the purpose of the edit box until the name of the edit box is not being announced by the VoiceOver.

### Environment
The VoiceOver user won’t be able to know the purpose of the edit box until the name of the edit box is not being announced by the VoiceOver.
[VO4_FontSize_Preference.zip](https://github.com/Microsoft/workbooks/files/1990867/VO4_FontSize_Preference.zip)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS4.2.1
Xamarin-Workbook-MAC
Wipro

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1825
